en/docs/setup/migration-guide.md

```diff
@@ -654,16 +654,48 @@ Follow the steps below to perform the upgrade.
           EXTENTSIZE 4;
     ```
 
-1.  Do the following database updates:  
+1.  If you manually added any custom OSGI bundles to the
+    `          <OLD_IS_HOME>/repository/components/dropins         `
+    directory, copy those to the
+    `          <NEW_IS_HOME>/repository/components/dropins         `
+    directory.
+2.  If you manually added any JAR files to the
+    `           <OLD_IS_HOME>/repository/components/lib          `
+    directory, copy those and paste in the
+    `           <NEW_IS_HOME>/repository/components/lib          `
+    directory.
+
+3.  Copy the `           .jks          ` files from the
+    `           <OLD_IS_HOME>/repository/resources/security          `
+    directory and paste in the
+    `           <NEW_IS_HOME>/repository/resources/security          `
+    directory.
+
+4.  If you have created tenants in the previous WSO2 Identity Server
+    version and if there are any resources in the
+    `          <OLD_IS_HOME>/repository/tenants         ` directory,
+    copy the content to the
+    `          <NEW_IS_HOME>/repository/tenants         ` directory.
+5.  If you have created secondary user stores in the previous WSO2 IS
+    version, copy the content in the
+    `           <OLD_IS_HOME>/repository/deployment/server/userstores          `
+    directory to the
+    `           <NEW_IS_HOME>/repository/deployment/server/userstores          `
+    directory.
+    
+    !!! warning
+        If you are migrating from WSO2 Identity Server 5.8.0 skip steps 6, 7, 8 and directly go to step 9. 
```

Comment:
Instead saying "If you are migrating from WSO2 Identity Server 5.8.0 skip steps 6, 7, 8 and directly go to step 9. " lets say "only if customer is on a version prior to 5.8.0, then do 6-9 on a 5.8.0 distribution.